[Add education component]Add education component in city file

zhou-1 opened the pull request:
Education component includes 3 parts with image shows.

gallettilance reviewed (changes requested):
Why did you change the `city.py` file? You should implement your functions in their respective files.

zhou-1 commented:
> Why did you change the `city.py` file? You should implement your functions in their respective files.

I thought we can "implement" it whatever we want, I will make a change...

zhou-1 commented:
Just deleted the images and changed back to original city file. Also added school and library to city.

gallettilance commented:
@zhou-1 the images are great actually please feel free to use them. I just ask that you implement that inside their respective files instead of in `city.py`.

zhou-1 commented:
> @zhou-1 the images are great actually please feel free to use them. I just ask that you implement that inside their respective files instead of in `city.py`.

@gallettilance I see, just added images and modified the function a little bit. Hope this will make it better. :)

gallettilance commented on 01-python/education/library.py:
```diff
@@ -1,3 +1,31 @@
+
 def draw_library():
-    print("library not found")
+img = Image.open("library.png")
```
needs an indent

zhou-1 replied:
I don't know why, but they are all looking good in my editor...

gallettilance commented on 01-python/education/library.py:
```diff
@@ -1,3 +1,31 @@
+	time.sleep(3)
+	im.close()
+	
+    print("+---------------------------------------------------------+")
```
this indent is also off

gallettilance commented on 01-python/education/school.py:
```diff
@@ -1,3 +1,31 @@
+	time.sleep(3)
+	im.close()
+	
+    print("+---------------------------------------------------------+")
```
this indent is off

gallettilance commented on README.md:
```diff
@@ -6,4 +6,6 @@ Code, slides, and notes for Boston University CS506 Fall 2020
 
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
\ No newline at end of file
+The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
```
please remove these changes

zhou-1 replied:
I will do that.

gallettilance commented on README.md:
```diff
@@ -6,4 +6,4 @@ Code, slides, and notes for Boston University CS506 Fall 2020
 
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
\ No newline at end of file
+Finish exercise 3
```
It's the opposite that is needed - you should not add the exercise 3 work here.

zhou-1 replied:
My bad, changed!

gallettilance commented on README.md:
```diff
@@ -3,7 +3,3 @@
-
-___
-
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
```
please add these deleted lines back so that you are not changing the readme when this merges

gallettilance commented:
also please squash your commits

zhou-1 reviewed (commented):
> also please squash your commits

@gallettilance Not sure whether I did it correctly, I tried " git rebase -i --root" ("git rebase -i HEAD~N" doesn't work for me)....

gallettilance commented on e -i --root:
```diff
@@ -0,0 +1,47 @@
+[33mcommit 365d21091e7d6d3262f9ff185b49546907a7dd70[m[33m ([m[1;36mHEAD -> [m[1;32mcomponent-branch[m[33m, [m[1;31morigin/component-branch[m[33m)[m
```
please remove this file

gallettilance commented on README.md:
```diff
@@ -4,6 +4,6 @@
 Code, slides, and notes for Boston University CS506 Fall 2020
 
-___
+___	
```
please remove this change

zhou-1 replied:
I don't know why this happened, I changed.

gallettilance reviewed (changes requested):
Great work! Just need to remove that strange root file and I will approve your PR

zhou-1 commented:
> Great work! Just need to remove that strange root file and I will approve your PR

@gallettilance I deleted it before I git add, commit and push; it is still here? :(

zhou-1 commented:
> Great work! Just need to remove that strange root file and I will approve your PR

@gallettilance I pushed again, hope this time it works!

zhou-1 commented:
Please let me know whether it works. :)
